gadgets/cmr/config/LandmarkDetection/gadgetron_cmr_landmark_detection_util.py

```diff
@@ -1,28 +1,87 @@
+"""Utility file for cmr landmark detection"""
+
+import os
 import sys
-import torch
-import torch.nn as nn
-import torch.optim as optim
-import torch.nn.functional as F
+import urllib.request
+import hashlib
 import numpy as np
-import scipy
 from scipy import ndimage
 
-def cpad_2d(data, RO, E1):
-    '''
-    data: [dRO, dE1, N], padded it round center to [RO, E1, N]
-    return value: (s_ro, s_e1), starting point of data in padded array
-    '''
+def is_model_valid(model_file, model_md5):
+    """Check whether the md5 hash is valid"""
+    
+    if not os.path.isfile(model_file):
+        return False
+
+    with open(model_file, "rb") as f:
+        file_hash = hashlib.md5()
+        while chunk := f.read(65536):
+            file_hash.update(chunk)
+        
+    return model_md5 == file_hash.hexdigest()
+
+
+def get_model(model_host, model_file, model_dest, model_md5):
+    """Get the model from the storage account.
     
+    Input:
+        model_host [str] : host address to download the model
+        model_file [str] : model file name
+        model_dest [str] : destination path to store the model
+        model_md5 [str] : model md5 hash
```

Comment:
I suggest we use SHA-256 instead of MD5, since the latter is usually a red flag because it is broken.

Reply:
Pushed a commit to use SHA-256.